src/Symfony/Component/HttpKernel/Tests/DependencyInjection/RegisterControllerArgumentLocatorsPassTest.php

```diff
@@ -545,9 +543,14 @@ protected function barAction(ControllerDummy $bar)
     }
 }
 
-class ContainerAwareRegisterTestController implements ContainerAwareInterface
+class ContainerAwareRegisterTestController
 {
-    use ContainerAwareTrait;
+    protected ?ContainerInterface $container;
+
+    public function setContainer(ContainerInterface $container = null): void
```

Comment:
is it still called?

Reply:
It is actually called in the pass:

https://github.com/symfony/symfony/blob/52a92926f7fed15cdff399c6921100a10e0d6f61/src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php#L80

I guess this should be kept regardless, as it was added when some checks on `ContainerAwareInterface` were removed in this commit https://github.com/symfony/symfony/commit/739789f384cca3b06a3b5f35793e116fff337473#diff-12c7c16828a1595f567c3a7e70a5678ee892171843c304b98a7760f6af1b2903